### DISCUSSION TITLE
Table relationship

### DISCUSSION THREAD

**Discussion by usamamumtaz1** [Budibase/budibase#3852] [Help]
Hi, 
i m  new here and also to coding. i m trying to create table vey simple take e.g Mentor and Profile

Mentor
MantorName
ManteeName

Profile
MyName
YourName

Issue
i want MyName to pull value from MentorName and Yourname to pull from ManteeName field

i have treid creating columns with relationship type but it always pull the name column(Mentee) in both fields , 

Can someone help?

**Comment by mjashanks** [comment]
Hi There.

In Budibase you must choose a single "Display Name" for your tables. Therefor, it's not currently possible to do exactly as you ask.

However. Is there a reason why you would not have the data model as below?

**Mentor**
MantorName : Text

**Profile**
MyName : Text 
Mantor : relationship to Mentor (displays Mentor Name)

Thus, a "Profile"  row would display the MantorName and the ManteeName
